### ISSUE TITLE
Validator link is down

### ISSUE THREAD

**Issue by ross-spencer** [w3c/activitystreams#519]
Please Indicate One: 
 * [ ] Editorial
 * [ ] Question
 * [ ] Feedback
 * [ ] Blocking Issue
 * [x] Non-Blocking Issue

Please Describe the Issue:

Coming from the [W3C docs](https://www.w3.org/TR/activitystreams-core/) and then checking out the GitHub, I see that the validator link is down. Looking at archive.org, likely since 2018? https://web.archive.org/web/*/https://as2.rocks/ There's one related issue too. It'd be good to have some information about this and its implications for folks, and alternative options? 

* References to as2.rocks validator: https://github.com/w3c/activitystreams/search?q=https%3A%2F%2Fas2.rocks%2F
* Related to: https://github.com/w3c/activitystreams/issues/411

**Comment by sebilasse** [comment]
Yes, we informed the responsible person 4 years ago but nobody else got access to the domain. 
See also https://socialhub.activitypub.rocks/t/https-test-activitypub-rocks-certifiicate-has-expired/2233/4

@cpmoser did https://github.com/yuforium/activity-streams-validator and maybe APdebug will help too: 
https://socialhub.activitypub.rocks/t/student-dev-making-the-first-steps-with-activitypub-and-doing-a-video/2401/4

**Comment by drazisil** [comment]
https://as2.rocks/ has now been claimed by an unrelated entity and should be removed to prevent possible damage from hostile actors.

**Comment by akuckartz** [comment]
@drazisil Can you please raise a new issue for that problem?

BTW: https://as2.rocks/ (ending with slash) and https://as2.rocks (without a slash at the end) are forwarded to different places.

**Comment by drazisil** [comment]
> @drazisil Can you please raise a new issue for that problem?
> 
> BTW: https://as2.rocks/ (ending with slash) and https://as2.rocks (without a slash at the end) are forwarded to different places.

I don't quite follow. Are you saying that as2.rocks is a silent proxy? Otherwise it's the same server.

**Comment by evanp** [comment]
I lost access to this domain a few years ago through neglect. I will see what I can do to get it back; otherwise, I agree we should remove it from the documentation or add an erratum.

It would be interesting to set up a validator again.  I think I have the code still available, so it might be possible to set up the validator on a different, more permanent, domain.
